x-pack/plugins/infra/server/lib/alerting/metric_threshold/lib/get_elasticsearch_metric_query.ts

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
Nit: This file is named after the function which feels a little and redundant and specific. Do you think we should be a little more generic? Maybe `metric_query.ts` is enough since it's already in the `metric` threshold dir?